### ISSUE TITLE
Plugin is with executable stack.

### ISSUE THREAD

**Issue by KottV** [jpcima/ysfx#57]
Hi,

I'm not sure is it a real problem or not, but openSUSE build server complains on building that .so has a executable stack. I think it was started after https://github.com/jpcima/ysfx/commit/b1921b6d34e720982ad3ba98c4ce5c7d494e75fd

The build log: https://build.opensuse.org/package/live_build_log/home:kill_it:JUCE/ysfx/openSUSE_Tumbleweed/x86_64
The error message:
```
[  165s] lv2-ysfx.x86_64: E: executable-stack (Badness: 10000) /usr/lib64/lv2/ysfx.lv2/ysfx.so
[  165s] vst-ysfx.x86_64: E: executable-stack (Badness: 10000) /usr/lib64/vst/libysfx.so
[  165s] vst3-ysfx.x86_64: E: executable-stack (Badness: 10000) /usr/lib64/vst3/ysfx.vst3/Contents/x86_64-linux/ysfx.so
[  165s] The binary declares the stack as executable. Executable stack is usually an
[  165s] error as it is only needed if the code contains GCC trampolines or similar
[  165s] constructs which uses code on the stack. One common source for needlessly
[  165s] executable stack cases are object files built from assembler files which don't
[  165s] define a proper .note.GNU-stack section.
```

**Comment by jpcima** [comment]
Fixed in c1d1d58

**Comment by KottV** [comment]
Thanks!
